public/docs/ts/latest/cookbook/a11y.jade

```diff
@@ -0,0 +1,1087 @@
+  
+  It integrates with browsers and automated testing tools, making this tool extremely useful in catching 
+  a variety of `a11y` violations before release. 
+    
+  `aXe` has browser extensions for both `Chrome` and `Firefox`, but as we looked at `Chrome` in the previous section, we
+  will use `Firefox` in our examples here.
+  
+  Once installed and activated, you can find `aXe` integrated into your browser's developer tools:
+
+figure.image-display
+  img(src="/resources/images/cookbooks/a11y/AXE-Overall-Analyze.png" alt="aXe integrated into browser development tools")  
+  
+:marked
+  All you need to do is click on the `Analyze` button and you will see a list of any `a11y` violations found:
+  
+figure.image-display
+  img(src="/resources/images/cookbooks/a11y/AXE-Overall-Fails.png" alt="aXe a11y violations summary")
+ 
+:marked
+  Click on these violations to drill down further and `aXe` will provide you with useful information on the error, 
+  how to find it in your `HTML` and guidelines to help you fix it:
+  
+figure.image-display
+  img(src="/resources/images/cookbooks/a11y/AXE-Details-Fails.png" alt="aXe a11y violation detail")  
+
+:marked
+  You can rerun aXe and any point during your development and it will show you when there are no `a11y` 
+  violations left:
+  
+figure.image-display
+  img(src="/resources/images/cookbooks/a11y/AXE-Overall-Pass.png" alt="aXe not a11y violations found")
+
+:marked
+  The message in this screenshot is an important one and again stresses that no one automated tool will ensure
+  full `a11y` compliance. And this brings us to another extremely important aspect of `a11y` testing.
+  
+:marked
+  ### The screen reader  
+  
+  When creating web pages it would be preposterous to never open it in a browser before release day. 
+  
+  The same is true for screen readers. The tooling already described should be used to speed up our development
+  cycle, but the only way to truly experience our web pages in a screen reader is by using one.
+  
+  However, using a screen reader for the first time can be a daunting task so here we look at three of the most
+  used screen reader / browser combinations and how to get started with them. You do not have to be an expert in
+  screen reader usage to test your websites for `a11y`!
+  
+.l-sub-section
+  :marked
+    You will use the skills you gained in the `keyboard navigation` section when using a screen reader. First get 
+    comfortable with keyboard-only navigation before adding a screen reader to it. This will help reduce the 
+    confusion that first time users of screen readers usually get confronted by.
+
+.l-sub-section
+  :marked
+    You will not get the same experience in every `screen reader / browser combination`, so do not spend too much time
+    trying to fix *"errors"* in combinations not mentioned here. Often you will be trying to fix a technology 
+    integration issue and not a problem on your page. Focus on the three combinations
```

Comment:
Sorry about the late response. I think the wording around the browser support recommendations is probably OK as-is. I just worry about offering that sort of guidance without having a source to back it up. 

Re. WCAG... I think you're right that the material is pretty imposing and could be off-putting. So I don't disagree about not linking to it immediately as another source of information. That said, I almost would like to see WCAG woven a bit more into the cookbook, as I think a lot of people interested in this subject will be looking for info specifically on how to meet WCAG in angular (because of organizational policy, for instance). But that probably has the same risk of alienating newcomers, so maybe it's just something to consider for the future if other folks ask for it.